### PR TITLE
add environment-independent maven wrapper

### PR DESCRIPTION
**Pull request by nkonev**
I added maven wrapper with fixed version for ability to repeatable build & test it independently installed maven version on contributor's machine.

So any new contributor can test it by
```bash
./mvnw clean test
```

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -263,7 +264,7 @@
 
     <properties>
         <jdk.version>6</jdk.version>
-        <jdk>${env.JAVA_HOME_6}</jdk>
+        <jdk>${env.JAVA_HOME_8}</jdk>
```
Comment (vladmihalcea) [inline pom.xml]: Why is this change needed for the Maven wrapper?

Comment (nkonev) [reply pom.xml]: Because threre isn' t required to install jdk 6 - tests can be run on jdk 8.

**Comment by nkonev** [comment]
@vladmihalcea can yoy merge it ? I reverted java version.

**Comment by nkonev** [comment]
Hello, can you left any feedback?

**Comment by vladmihalcea** [comment]
@nkonev I won't have time to review this issue since I'm working on my Hypersistence Optimizer project now.

Anyway, this is a minor issue from my perspective, just like the Docker Compose one since it does not impact the clients of the framework.

Thanks for your contribution.

**Comment by vladmihalcea** [comment]
Applied upstream, thanks
